service/src/main/java/bio/terra/cbas/dependencies/github/GitHubClient.java

```diff
@@ -24,14 +24,23 @@ public RepoInfo getRepo(String organization, String repo, String token)
       throws GitHubClientException {
 
     WebTarget target = client.target(BASE_URL).path("/repos").path(organization).path(repo);
-    Response response =
-        target
-            .request(MediaType.APPLICATION_JSON_TYPE)
-            .header("Accept", "application/vnd.github+json")
-            .header("Authorization", "Bearer " + token)
-            .header("X-GitHub-Api-Version", "2022-11-28")
-            .get();
-
+    Response response;
+    if (token.isEmpty()) {
+      response =
+          target
+              .request(MediaType.APPLICATION_JSON_TYPE)
+              .header("Accept", "application/vnd.github+json")
+              .header("X-GitHub-Api-Version", "2022-11-28")
+              .get();
+    } else {
+      response =
+          target
+              .request(MediaType.APPLICATION_JSON_TYPE)
+              .header("Accept", "application/vnd.github+json")
+              .header("Authorization", "Bearer " + token)
+              .header("X-GitHub-Api-Version", "2022-11-28")
+              .get();
+    }
```

Comment:
TOL: this can be made less repetitive by extracting the common bits out of the if..else block and the authorization header to be added only in the case token is empty